app/src/main/java/com/xmartlabs/taskloans/ui/screens/dashboard/tabs/TeamFragment.kt

```diff
@@ -31,9 +30,9 @@ class TeamFragment : BaseViewBindingFragment<FragmentTeamBinding>() {
     loadTeamList()
   }
 
-  override fun onDestroy() = withViewBinding {
-    super.onDestroy()
-    teamRecyclerView.adapter = null
+  override fun onDetach() {
+    super.onDetach()
+    adapter = null
```

Comment:
this is an error, roll back the change. As feli said, you will add a memory leak